akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -106,24 +139,18 @@ class IntroSpec extends ActorTestKit with TypedAkkaSpecWithShutdown {
 
   "Hello world" must {
     "say hello" in {
-      // TODO Implicits.global is not something we would like to encourage in docs
       //#hello-world
-      import HelloWorld._
-      // using global pool since we want to run tasks after system.terminate
-      import scala.concurrent.ExecutionContext.Implicits.global
 
-      val system: ActorSystem[Greet] = ActorSystem(greeter, "hello")
+      val system: ActorSystem[HelloWorldMain.Start] =
+        ActorSystem(HelloWorldMain.main, "hello")
 
-      val future: Future[Greeted] = system ? (Greet("world", _))
+      system ! HelloWorldMain.Start("World")
+      system ! HelloWorldMain.Start("Akka")
 
-      for {
-        greeting ← future.recover { case ex ⇒ ex.getMessage }
-        done ← {
-          println(s"result: $greeting")
-          system.terminate()
-        }
-      } println("system terminated")
       //#hello-world
+
+      Thread.sleep(500) // it will not fail if too short
```

Comment:
Could change this to shut its self down then await termination of the system

Reply:
That would introduce even more code and concepts in the example code. This is outside the doc snippet. Also, I don't want to give the first impression that an ActorSystem is supposed to be used for a single task and then shutdown immediately.